app/src/main/java/cz/muni/fi/pv256/movio2/uco_410434/ui/FilmDetailActivity.java

```diff
@@ -0,0 +1,45 @@
+package cz.muni.fi.pv256.movio2.uco_410434.ui;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.view.MenuItem;
+
+import cz.muni.fi.pv256.movio2.uco_410434.R;
+
+public class FilmDetailActivity extends AbstractActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_film_detail);
+
+        // Show the Up button in the action bar.
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true);
+        }
+
+        if (savedInstanceState == null) {
+            Bundle arguments = new Bundle();
+            arguments.putParcelable(FilmDetailFragment.ARG_ITEM,
+                    getIntent().getParcelableExtra(FilmDetailFragment.ARG_ITEM));
+
+            FilmDetailFragment fragment = new FilmDetailFragment();
```

Comment:
Fragmenty sa vacsinou vytvaraju pomocou statickej metody ktora ma ako parameter "arguments". myslim ze to tak ma aj Jonas v priklade